src/pages/ExamsPage/hooks.js

```diff
@@ -172,6 +172,23 @@ export const useCreateAllowance = () => {
   };
 };
 
+export const useEditAllowance = () => {
```

Comment:
Note: I created a `useEditAllowance` hook which is very similar to the `useCreateAllowance` hook (uses the same post API endpoint), but this one doesn't create separate form data using the `createAllowanceData` util function.

I am now wondering if it is better for this to take fields separately rather than as a form (so username, examId, and extraTimeMins)... Can refactor if this is better.

Reply:
I think I addressed this on your question about the `payload` above, but let me know if you'd like more feedback.